cpp/include/cugraph/graph_functions.hpp

```diff
@@ -1052,4 +1052,60 @@ remove_multi_edges(raft::handle_t const& handle,
                    std::optional<rmm::device_uvector<edge_type_t>>&& edgelist_edge_types,
                    bool keep_min_value_edge = false);
 
+/**
+ * @brief Shuffle external vertex ids and values to the proper GPU.
+ * *
```

Comment:
Better document the supported value types.

And sorry for nitpicking, but it might be more natural to list `shuffle_external_vertices` first and `shuffle_external_vertices_and_values` second.

Reply:
And `vertex_value_pairs` is more consistent than `vertices_and_values` in our codebase.

Reply:
Changed.